### ISSUE TITLE
md-radio-button: If ng-model value is null and ng-value is used, screen readers voice the last option first

### ISSUE THREAD

**Issue by calebegg** [angular/material#5856]
Demo:

http://codepen.io/anon/pen/EVJWGm

Here's what I think happens: before `ng-value` can fire, changing the `value=""` attribute, `md-radio-button` decides that the `null` `ng-model` value means that the radio button with `ng-value` is selected (since it has no `value=""` attr). Then, `ng-value` fires, updating `value` and deselecting that option. But the damage has been done somehow, and now from the screen reader's perspective, the button with `ng-value` was selected and then deselected and so should be read first.

I worked around it by initializing `value=` to something else (like -1) that was never a valid `ng-model` value.
